.github/workflows/test_examples.yml

```diff
@@ -18,12 +19,20 @@ jobs:
       GH_SERVICE_ACCOUNT_NAME: "tflite-bot"
       GH_SERVICE_ACCOUNT_EMAIL: "tflite-bot@antmicro.com"
       GH_SERVICE_ACCOUNT_TOKEN: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+      GOOGLE_APPLICATION_CREDENTIALS: "/tmp/google_sa.json"
```

Comment:
can we have no `tmp`? Do we have to store a file?

Reply:
Unfortunately, we do have to store the file; Google's authentication library requires it.

Reply:
I got rid of it by using the `runner` context variables -- the `${{ runner.temp }}` expression always resolves to a temporary directory of a runner (in a platform-agnostic way).

In case you find my implementation slightly weird, it's because I had to use some trickery, as it doesn't seem to be possible to use this variable when setting job-wide variables.